docs/devel/reporting.rst

```diff
@@ -30,12 +30,12 @@ and lists all contributors to a given language:
 
     * Dutch
 
-        * Jane Doe <jane@example.com> (42)
+       * Jane Doe <jane@example.com> (42)
```

Comment:
@orangesunny 
Ref. https://github.com/WeblateOrg/weblate/commit/502660952d78a97ffd765eb63c271bf93cacaf39
"Roe" is the localised Dutch variant (same as German) https://nl.wikipedia.org/wiki/John_Doe

The idea is to be good at localisation, and that the same person (given by the identical e-mail) wouldn't be likely to edit both Dutch and Czech. Good examples are not lazy?

Reply:
Using localized _Does_ would be interesting, but only consistently throughout the whole docs, in localized version (other languages) of the docs. In this case (across the English docs version), one name is correct across all languages exampled in the reports' snippet; Jane Doe is clearly a productive translator as she contributed to multiple languages. Her name should not be translated to assure she is the same person.